### DISCUSSION TITLE
How to set the minimum width of a Window Apps ?

### DISCUSSION THREAD

**Discussion by vituthanh** [unoplatform/uno#18862] [Q&A] [answered]
I don't know of any property to set the minimum width of a Windows app. Please help me.

**Comment by MartinZikmund** [comment, marked as answer]
This is an ongoing feature request on WinAppSDK - https://github.com/microsoft/microsoft-ui-xaml/issues/7296. On the WinUI target you can use the WinUIEx library to add this capability more easily. In case of the `-desktop` target, we don't directly surface support for this yet, but I created https://github.com/unoplatform/uno/issues/18863 to add a uno-specific API for that. Follow this issue for progress
